src/Symfony/Component/Translation/Bridge/Crowdin/Tests/CrowdinProviderTest.php

```diff
@@ -382,7 +377,7 @@ public function testReadForOneLocaleAndOneDomain(string $locale, string $domain,
             'headers' => [
                 'Authorization' => 'Bearer API_TOKEN',
             ],
```

Comment:
The provider instantiation should be done with `$this->createProvider`, and include a `/project/PROJECT_ID/` at the end of base URI. (same in all file)

Reply:
Tests pass because the implementation doesn't care about the base URI, that's the point. We don't need to test with the real base URI.